shared/chat/conversation/info-panel/menu/container.js

```diff
@@ -0,0 +1,60 @@
+// @flow
```

Comment:
Ah, is this implementing the gear icon? This is another ticket I believe, so you're doing 2 for 1!